ariadne/transformations.py

```diff
@@ -175,17 +150,21 @@ def __call__(self, data):
         # Returns:
             data (pd.DataFrame): transformed dataframe
         """
-        # assert type(data) == pd.core.frame.DataFrame, "unsupported data format"
-        data = super().drop_fakes(data)
+        fakes = data.loc[data[self.track_column] == -1, :]
+        data = self.drop_fakes(data)
```

Comment:
maybe we can combine these lines. `drop_fakes` may return two arguments - cleaned data and the removed fakes. I don't want to split the logic in a such way